packages/js/src/addon-installation-successful.js

```diff
@@ -0,0 +1,14 @@
+import { render } from "@wordpress/element";
+import AddonInstallationSuccessful from "./components/AddonInstallationSuccessful";
+
+const elementToInsert = document.createElement( "div" );
+elementToInsert.setAttribute( "id", "wpseo-app-element-2" );
+
+// Insert under the first heading on the page.
+const elementToAppend = document.getElementsByClassName( "yoast-heading-highlight" )[ 0 ].parentElement;
```

Comment:
Have you considered adding an ID to the heading we want to attach this to?
Would make changes to the layout of the page in the future a bit more easy.

Reply:
Did consider, probably should've considered it a bit more. done as suggested.